src/main/java/com/crud/tasks/controller/TrelloController.java

```diff
@@ -0,0 +1,29 @@
+package com.crud.tasks.controller;
+
+import com.crud.tasks.domain.TrelloBoardDto;
+import com.crud.tasks.trello.client.TrelloClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("/v1/trello")
+public class TrelloController {
+
+    @Autowired
+    private TrelloClient trelloClient;
+
+    @RequestMapping(method = RequestMethod.GET, value = "getTrelloBoards")
+    public void getTrelloBoards() {
+
+        List<TrelloBoardDto> trelloBoards = trelloClient.getTrelloBoards().stream()
+                .filter(board -> board.getId()!=null&&board.getName().contains("Kodilla"))
```

Comment:
czytelniej bedzie rozbic to na 2 filtry 
* jeden dla id != null
* drugi dla contains("...")

Reply:
"Kodilla" powinna byc stala `private static final ..`